apps/rush-lib/src/logic/VersionManager.ts

```diff
@@ -90,6 +95,59 @@ export class VersionManager {
       });
       changeManager.updateChangelog(!!shouldCommit);
     }
+
+    this._updateTarballIntegrities();
+  }
+
+  private _updateTarballIntegrities(): void {
```

Comment:
When in workspaces, tarball integrities will not need to be updated but version specifiers do need to be updated. Rushstack is now on workspaces so the lockfile format can be viewed in pnpm-lock now :)

Reply:
I think it might be okay to leave version specifiers as is. Let me verify.